src/main/java/com/mitchej123/hodgepodge/mixins/early/minecraft/MixinLongHashMap.java

```diff
@@ -0,0 +1,64 @@
+package com.mitchej123.hodgepodge.mixins.early.minecraft;
+
+import net.minecraft.util.LongHashMap;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Overwrite;
+import org.spongepowered.asm.mixin.Unique;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import it.unimi.dsi.fastutil.longs.Long2ObjectOpenHashMap;
+
+@Mixin(LongHashMap.class)
```

Comment:
Imo it'd be better to make a `public class FastUtilsLongHashMap extends LongHashMap`, override all methods and just replace the `new LongHashMap()` with `new FastUtilsLongHashMap()`.

Reply:
Also thermos has its own, worth considering to look for collisions: https://github.com/GTNewHorizons/Thermos/blob/1.7.10/src/main/java/thermos/wrapper/LongHashMapTrove.java

Reply:
Seems reasonable, I wasn't really sure what other places it would be used in so I figured globally replacing this class entirely would cover more cases